eng/Publishing.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <Target Name="_AddFilePackagesToPublish" AfterTargets="Publish" Condition="'@(PackagesToPublish)' == ''">
```

Comment:
Is `Publishing.props` a reserved filename that gets evaluated automatically when it exists? Or does this target get au automatically evaluated simply by linking it to the "Publish" target via "AfterTargets"?

Reply:
`Publishing.props` is an arcade extensibility point for participating in the publish process.  
https://github.com/dotnet/arcade/blob/812d978c303174dc1aa305d7359e79053d7d4971/Documentation/ArcadeSdk.md?plain=1#L296